monkey/monkey_island/cc/ui/src/components/map/MapOptions.js

```diff
@@ -50,7 +52,8 @@ export const basic_options = {
   }
 };
 
-export function getOptions(stateList) {
+export function getOptions() {
+  let stateList = Object.keys(NodeGroup).filter(key => !isNaN(Number(NodeGroup[key])))
```

Comment:
I don't understand what's the point of this?

Reply:
Let's wrap it in a well-named function that explains what it does.